pf_driver/src/pf/pf_interface.cpp

```diff
@@ -98,7 +93,7 @@ bool PFInterface::start_transmission()
 
         std::string host_ip = transport_->get_host_ip();
         port_ = transport_->get_port();
-        info_ = protocol_interface_->request_handle_udp(host_ip, port_);
+        info_ = protocol_interface_->request_handle_udp(host_ip, port_, pkt_type);
```

Comment:
The API was changed in previous commit https://github.com/PepperlFuchs/ROS_driver/pull/26/commits/4ad6e9e9c6bbbd12ccc55385545b74df66216f49. It would be better to make that API change here to keep it together. It is a bit confusing the current way.

Reply:
I kinda liked it this way, but it's your repo, so... change incoming.

Reply:
> but it's your repo, 

But I am always to new ideas :)